sdk/identity/identity/test/node/clientCertificateCredential.spec.ts

```diff
@@ -65,7 +65,7 @@ describe("ClientCertificateCredential", function() {
       );
 
       const queryParams = qs.parse(authRequest.body);
-      const jwtToken = jws.decode(queryParams.client_assertion);
+      const jwtToken = jws.decode(queryParams.client_assertion as any as string);
```

Comment:
What is the reason for this casting? I understand that `jws.decode()` takes a string, but why is it that we get an error now, but not when the last stable version of keyvault keys was released?

Reply:
I don't know. I know that as we go back in time, stability on the commits to master is lost, due to how the environment changes. By environment I mean how the CI pipelines have changed. Only recently we've been enforcing stuff more consistently across the whole project.

There's also the fact that this change does absolutely no damage to the keyvault hotfix release.

Reply:
Just to avoid loose threads, the way core-lro could be fixed for the identity release was to move it forward to the next stable release, only in that folder, while downgrading a couple of dependencies. Fortunately, identity didn't depend on core-lro, so that was fine.

Reply:
Ok, but `as string` should work just fine right? Why do we need to cast to any and then cast to string?

Reply:
Just `as string` was enough indeed! Nice catch!